save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/DemoManagerController.kt

```diff
@@ -179,26 +212,44 @@ class DemoManagerController(
         description = "Get demo info.",
     )
     @ApiResponse(responseCode = "200", description = "Successfully fetched demo status.")
+    @ApiResponse(responseCode = "403", description = "Not enough permission for accessing given project.")
+    @ApiResponse(responseCode = "404", description = "Could not find project in organization.")
```

Comment:
the same: here we get return 404 if demo is not found

Reply:
Done in [4a36dc6](https://github.com/saveourtool/save-cloud/pull/1688/commits/4a36dc64bb2f0c58be27a075e672c98364c1daac)